src/content/en/fundamentals/instant-and-offline/pwa-architecture/index.md

```diff
@@ -0,0 +1,922 @@
+    <td>
+    <a href="https://app.jalantikus.com/">https://app.jalantikus.com/</a>
+    <a href="https://m.geo.tv/">https://m.geo.tv/</a>
+    <a href="https://app.kompas.com/">https://app.kompas.com/</a>
+    <a href="https://www.nfl.com/now/">https://www.nfl.com/now/</a>
+    <a href="https://www.chromestatus.com">https://www.chromestatus.com</a>
+ </td>
+  </tr>
+  <tr>
+    <td>Publishing</td>
+    <td>AppShell + SSR content for entry pages</td>
+    <td>
+    <a href="https://react-hn.appspot.com">https://react-hn.appspot.com</a> 
+ </td>
+  </tr>
+  <tr>
+    <td>Publishing</td>
+    <td>Streams for body content / UI</td>
+    <td><a href="https://wiki-offline.jakearchibald.com/wiki/The_Raccoons">https://wiki-offline.jakearchibald.com/wiki/The_Raccoons</a></td>
+  </tr>
+  <tr>
+    <td>Social</td>
+    <td>AppShell</td>
+    <td><a href="https://web.telegram.org/">https://web.telegram.org/</a></td>
+  </tr>
+  <tr>
+    <td>E-commerce</td>
+    <td>Application Shell</td>
+    <td>
+    <a href="https://housing.com">housing.com</a>
+    <a href="https://m.aliexpress.com/">https://m.aliexpress.com/</a>
+    <a href="https://kongax.konga.com/">https://kongax.konga.com/</a>
+    <a href="https://m.flipkart.com">https://m.flipkart.com (mobile/emulate)</a>
+    <a href="https://m.airberlin.com/en/pwa">https://m.airberlin.com/en/pwa</a>
+ </td>
+  </tr>
+  <tr>
+    <td>E-commerce</td>
+    <td>AppShell + SSR content for entry page</td>
+    <td>
+    <a href="https://selio.com/">https://selio.com (try on mobile/emulate)</a>
+    <a href="https://lite.5milesapp.com/">https://lite.5milesapp.com/</a>
+</td>
+  </tr>
+  <tr>
+    <td>Conference</td>
+    <td>AppShell</td>
+    <td>
+    <a href="https://events.google.com/io2016/schedule">https://events.google.com/io2016/schedule</a></td>
+  </tr>
+</table>
+
+## Migrating an Existing Site to PWA {: #migrating-an-existing-site-to-pwa }
+
+When migrating to a PWA, there are no hard and fast requirements around what to cache. You might, in fact, find it useful to think of your offline strategy as a series of milestones. It is feasible to begin by adding a simple service worker and just caching static assets, such as stylesheets and images, so these can be quickly loaded on repeat visits. 
+
+The next step might be caching the full-page HTML or caching the app shell to serve the empty UI first, and then allow the data layer to be pulled in from the server. This is the equivalent of a server sending down a rendered page without any results. 
+
+Each milestone allows you to deploy separately, measure the potential performance gains from each step you explore, and progressively roll out a better PWA. 
+
+Note: Understanding the network traffic is key to successful migration. You can use the guidelines in [Measure Resource Loading Times](https://developers.google.com/web/tools/chrome-devtools/profile/network-performance/resource-loading) to get started using the Network DevTools panel.
```

Comment:
Don't use fully qualified URLs, instead use full urs. Specifically change the url here from https://developers.google.com/web/tools/... to /web/tools/...